applications/deltacast_transmitter/python/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+	"application": {
+		"name": "Videomaster transmitter example",
+		"authors": [
+			{
+				"name": "Pierre PERICK",
+				"affiliation": "DELTACAST"
+			}
+		],
+		"language": "Python",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "0.5.0",
+			"tested_versions": [
+				"0.5.0",
+				"2.9.0"
+			]
+		},
+		"platforms": [
+			"x86_64",
+			"aarch64"
+		],
+		"tags": [
+			"Deltacast",
+			"VideoMaster"
+		],
+		"ranking": 2,
+		"dependencies": {},
```

Comment:
Suggest noting the Deltacast / Videomaster library dependency and version in metadata